obd/commands.py

```diff
@@ -279,6 +280,27 @@
     OBDCommand("GET_CURRENT_DTC", "Get DTCs from the current/last driving cycle", b"07", 0, dtc, ECU.ALL, False),
 ]
 
+
+__mode9__ = [
+    #                      name                             description                            cmd     bytes       decoder       ECU        fast
+    OBDCommand("PIDS_9A"                    , "Supported PIDs [01-20]"                            , b"0900",  7, pid,                ECU.ALL,     True),
+    OBDCommand("VIN_MESSAGE_COUNT"          , "VIN Message Count"                                 , b"0901",  3, count,              ECU.ENGINE,  True),
+    OBDCommand("VIN"                        , "Get Vehicle Identification Number"                 , b"0902", 22, encoded_string(17), ECU.ENGINE,  True),
```

Comment:
Is "`Get `" needed?

Reply:
How can the subject progress?

Can we help?